.github/workflows/docker-publish.yml

```
@@ -0,0 +1,55 @@
+name: Docker
+
+on:
+  workflow_dispatch:
+    inputs:
+      tagInput:
+        description: 'Tag'
+        required: true
+
+  release:
+    types: [created]
+    tags:
+      - 'v*'
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Do we want to update this to the [latest](https://github.com/marketplace/actions/docker-setup-qemu)?
```suggestion
        uses: docker/setup-qemu-action@v3
```